snaps/index.mdx

```diff
@@ -32,9 +32,9 @@ The following Snaps features are available in the stable version of MetaMask:
     },
     {
       icon: require("./assets/features/notifications.png").default,
-      href: "reference/snaps-api#snap_notify",
+      href: "features/notifications",
       title: "Notifications",
-      description: "Notify users directly in MetaMask, or in their OS."
+      description: "Notify users directly in MetaMask, or natively in their browser."
```

Comment:
I do not know if it is correct to change the last word. Browser notification APIs are displayed in the OS. They are not displayed in the browser. The Chrome spec says they are displayed in the system tray: https://developer.chrome.com/docs/extensions/reference/api/notifications